account_asset_management/models/account_move.py

```diff
@@ -79,6 +80,10 @@ def post(self):
                     .with_context(create_asset_from_move_line=True, move_id=move.id)
                     .create(vals)
                 )
+
+                if analytic_account and not asset.account_analytic_id:
+                    asset.account_analytic_id = analytic_account
```

Comment:
Shouldn't the created asset use the analytic account explicitly defined in the aml/invoice? I think so.